app/models/federal_register_client.rb

```diff
@@ -89,4 +85,19 @@ def sorn_params(result)
       data_source: :fedreg
     }
   end
+
+  def update_agencies(result, sorn)
```

Comment:
would it make sense to have a test on this method?

Reply:
Oh, yeah we do.

Reply:
I added a test for that behavior.